### DISCUSSION TITLE
Telegram download multiple items

### DISCUSSION THREAD

**Discussion by mv0hra** [ferdium/ferdium-app#1174] [Ideas]
So in the official Telegram app you can select several items and save ALL items without having to keep clicking on 'Save', do you think this is something we can introduce in Ferdium?

**Comment by vraravam** [comment]
Is this possible via telegram web? If so, then it should be possible via Ferdium as well. If not, then no.

**Reply by mv0hra** [reply]
Sorry I did not realise that Ferdium was the equivalent to Telegram Web, I was looking for an alternative to the Telegram App for Windows.
This option is not possible in Telegram Web, but it is in the Telegram App for Windows.

**Comment by vraravam** [comment]
Also - please do not start discussion threads for such questions.

**Reply by mv0hra** [reply]
I thought this would have been the right place to post ideas. My bad.
